config/server.lua

```diff
@@ -116,5 +116,17 @@ return {
 
     giveVehicleKeys = function(src, plate)
         exports.qbx_vehiclekeys:GiveKeys(src, plate)
+    end,
+
+    hasKeys = function()
+        exports.qbx_vehiclekeys:HasKeys()
+    end,
```

Comment:
This is used by the client, but is marked as server config

Reply:
Fixed.